syntax/puppet.vim

```diff
@@ -1,167 +1,433 @@
-" puppet syntax file
-" Filename:     puppet.vim
-" Language:     puppet configuration file
-" Maintainer:   Luke Kanies <luke@madstop.com>
-" URL:
-" Last Change:
-" Version:
```

Comment:
Ok yup, you're right. It is a convention and it would be nice to have it, but nothing is better than wrong.